index.js

```diff
@@ -406,12 +415,20 @@ class HtmlWebpackInlineSVGPlugin {
      */
     isNodeValidInlineImage (node) {
 
-        return !!(
-            node.nodeName === 'img'
-            && ((this.inlineAll && _.filter(node.attrs, { name: 'inline-exclude' }).length === 0)
-                || _.filter(node.attrs, { name: 'inline' }).length)
-            && this.getImagesSrc(node))
+        const isInlineAllAndDoesNotHaveInlineExclude = (
+            this.inlineAll &&
+            _.filter(node.attrs, { name: 'inline-exclude' }).length === 0
+        )
+        const isInlineNotMarkedAsFailed = (
+            _.filter(node.attrs, { name: 'inline' }).length &&
+            !_.filter(node.attrs, { name: 'failed' }).length
```

Comment:
@AntonioRedondo on each iteration we're filtering the nodes that contains the `failed` attribute so we don't process them again if they failed when being fetched. This gets cleaned up at the end of the process.

Reply:
For consistency:
```suggestion
            _.filter(node.attrs, { name: 'failed' }).length === 0
```